manifests/params.pp

```diff
@@ -304,4 +297,10 @@
   $postgresql_conf_path = pick($postgresql_conf_path, "${confdir}/postgresql.conf")
   $recovery_conf_path   = pick($recovery_conf_path, "${datadir}/recovery.conf")
   $default_database     = pick($default_database, 'postgres')
+
+  $service_reload = if $facts['service_provider'] == 'systemd' {
+    "systemctl reload ${service_name}"
+  } else {
```

Comment:
Would it make sense to still guard this under `$facts['kernel']` or similar to catch BSDs?

Reply:
I can, but what should we do if it's neither systemd nor BSD? We currently don't have a case for this.

Reply:
I wanted to check FacterDB for Factsets from netBSD/OpenBSD, because I'm not sure what they report as `kernel` fact, but we don't have any Factsets for those :(